packages/docs/docs/fhir-datastore/profiles.md

```diff
@@ -168,3 +171,34 @@ to satisfy the field presence requirement, while also denoting why the data is m
 ```
 
 [data-absent-ext]: http://hl7.org/fhir/StructureDefinition/data-absent-reason
+
+## Updating Profiles
+
+Updating FHIR profiles is different than updating other resources in FHIR. The process is more similar to a database migration, and the profiled resources will need to be revalidated once the update is complete. This revalidation does not happen automatically, but will be checked the next time the resource is written to.
+
+This section offers some best practices to make updating profiles as smooth and painless as possible when using Mepdlum.
+
+When updating a profile, you should create a _new_ [`StructureDefinition`](/docs/api/fhir/resources/structuredefinition) resource for the updated profile. This will be similar to the original, but it will have the changes you've made, as well as an udpated `version` field. The `version` element is used to track changes to profile over time.
+
+Medplum does not currently fully implement versioning per the FHIR spec. Because of this, when updating a profile the version should also be included as a part of the url. This allows you to make changes to your profile without invalidating resources that do not yet comply to the updated profile.
+
+For example, say you have a patient profile that requires patients to have an associated email address. The url for this profile would be `https://example.com/profiles/foo-patient/1.0.0`. In this example `foo-patient` is the name of the profile and `1.0.0` is the version.
```

Comment:
Break this out into an `details` tag for the example. 

To make it easier to read, walkthrough the steps in bullet points